### PR TITLE
Degrees and decimal minutes - add parsing from and formatting to 

### PR DESCRIPTION
**Pull request by evdb**
Decimal minutes are also a common format to use.

Have bumped the version as part of this pull request too as I wanted to start using the code straight away myself.

### CODE REVIEW

**Comment by aeinbu** [comment]
Hi!

I think the correct version with this change should be 1.0.6. (I think the previous version no. in magellan.js was outdated to start with.)
Aslo, the change the version no. should also be reflected in the package.json file.

**Comment by evdb** [comment]
Have updated all the versions to 1.0.6 now.

I added a test to the suite to check that the version number in `package.json` is in sync too :)

File: test/suite.js [hunk 1]
```diff
@@ -20,7 +27,8 @@ assert.equal('12.345600', magellan(magellan(12.3456).toDMS(' ')).toDD())
 /* VERSION */
 
 // magellan must correctly expose its version
-assert.equal('1.0.2', magellan.version)
+assert.equal('1.0.6', magellan.version)
```
Comment (captivationsoftware) [inline test/suite.js]: I would nix this line altogether in favor of your package.json assertion below

**Comment by captivationsoftware** [comment]
Nice work! I'm going to probably bump this to version 1.1.0 because toDM is a new capability. Any objections?

**Comment by aeinbu** [comment]
Nope

**Pull request merged**